rsksmart/tokenbridge-ui
# [TBRE-512] - Remove Rinkeby from the options available in the Networks list available

### Title of pull request
Remove Rinkeby from the options available in the Networks list available

### Description

- Remove rinkeby network from list of available networks
- Remove rinkeby confs and properties

### How to Test

- Run the react-server
- check for rinkeby not being displayed
<img width="1159" alt="image" src="https://user-images.githubusercontent.com/100863009/158293883-79c91539-a861-4d35-9eb7-71aa64336f16.png">


### Checklist
- [] Lint is clean `npm run lint`
- [] Prettier is passing `npm run prettier`
